### ISSUE TITLE
[Bug]: Still overground biome - Fabric 1.20.1

### ISSUE THREAD

**Issue by xR4YM0ND** [HexagonNico/UndergroundJungle#10]
Same as https://github.com/HexagonNico/UndergroundJungle/issues/4

- Fabric 1.20.1
- Loader 0.15.3

- World Type: BetterX (from BetterNether & BetterEnd)
- Terrablender 3.0.0.169
- Expanded Ecosphere 3.2.3
- Continents 1.1.4
- Geophilic 2.1.0

## Description
Most of the time the caves work, but sometimes they reach till the top.

![2023-12-31_17 49 20](https://github.com/HexagonNico/UndergroundJungle/assets/97298005/4d2a4ab9-78bb-4710-a242-e5d9c8e34346)

![2023-12-31_17 49 37](https://github.com/HexagonNico/UndergroundJungle/assets/97298005/4a2c2dab-7f94-4f95-aa30-ed602c8739fd)

Deepest spot y 73

**Comment by HexagonNico** [comment]
Update on this problem:

I think this may be a problem on TerraBlender's end, because even if I increase the Depth parameter the biome still generates above ground. I've found it is mentioned in [this issue](https://github.com/Glitchfiend/TerraBlender/issues/108), but they're saying it is caused by parameters not being set correctly. Vanilla cave biomes have a Depth value of (0.2, 0.9), which is the same I use for the underground jungle and it works fine without TerraBlender.

I've found that if instead of adding the underground jungle as a new biome I replace the lush cave in the region then it will generate underground. Although this isn't ideal because it doesn't give much control over where the biome generates, I think this is what I will do for the next update.
